DependencyInjection/EkinoNewRelicExtension.php

```diff
@@ -129,6 +129,10 @@ public function load(array $configs, ContainerBuilder $container): void
             $container->setParameter('ekino.new_relic.application_name', $config['application_name']);
             $container->setAlias('ekino.new_relic.logs_handler', $config['monolog']['service'])->setPublic(false);
         }
+
+        if (!$config['enabled']) {
+            $container->removeDefinition(ExceptionListener::class);
```

Comment:
Regarding the logs, in the PR description, this is not the class responsible of the issue. The loop is inside the `DeprecationListener`.
Moreover this is not related to tests and the right way to fix.


You should try
```
try {
  return $prevErrorHandler ? $prevErrorHandler($type, $msg, $file, $line, $context) : false;
} catch (DeprecationException $e) {
}
````

Reply:
This was the original error:
```
Undefined offset: 0
```

I was accessing `[0]` on something that wasn't an array.

Reply:
Hm. we are not using any arrays in this class.. strange

Reply:
Sorry, I wasn't clear at all. This error is happening inside my `phpunit` tests:
```

1) Functional\Core\ApiBundle\GraphQL\Location\CreateLocationFunctionalTest::it_should_xxx
Undefined offset: 0

/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/vendor/sentry/sentry/lib/Raven/Breadcrumbs/ErrorHandler.php:34
/vagrant/vendor/sentry/sentry/lib/Raven/ErrorHandler.php:127
/vagrant/vendor/ekino/newrelic-bundle/Listener/DeprecationListener.php:41
/vagrant/tests/Functional/Core/ApiBundle/GraphQL/Location/CreateLocationFunctionalTest.php:92
```

This is the line:
```php
self::assertEquals('XXX', $response['data']['createLocation']['errors'][0]['code']);
```

Reply:
Oh, okey. So when you run the line `self::assertEquals('XXX', $response['data']['createLocation']['errors'][0]['code']);` when ` $response['data']['createLocation']['errors'][0]` is not defined, you get this massive (unrelated) error. 

I think this has something to do with how we register our custom error handler. Could we change that somehow?

Reply:
hmm. given it's not an infinite loop and the test ends-up with the phpunit failure. 

I've the filling that the issue is: the method `register` is called several time and the method `unregister` is never called. Which decorate the previous handler, etc.. It's an issue with PHPunit which boot things several times in a row.

Maybe the easy fix it to make `$isRegistered` static like in the symfony bridge https://github.com/symfony/symfony/blob/1fc66ff080d439634a1a7d5a09cd98896506a45c/src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php#L42

Reply:
Isn't the issue that this bundle never actually calls the `unregister` method on shutdown? If we would change `$isRegistered` to static, it wouldn't work because when running multiple functional tests, you boot- and shutdown the container for every test.

Reply:
could you have a try to 209 ?